greengrass_defender_agent/config.py

```diff
@@ -14,9 +14,16 @@
 MIN_INTERVAL_SECONDS = 300  # minimum sample interval at which metrics messages can be published
 SCHEDULED_THREAD = None
 SAMPLE_INTERVAL_CONFIG_KEY = "SampleIntervalSeconds"
+CONNECT_AND_PUBLISH_RETRY_CONFIG_KEY = "MaxConnectAndPublishRetry"
```

Comment:
Do I need to registry this key somewhere to make it legally reserved or can I just try fetch it directly?

Reply:
It's defined in the component's recipe. We'll add default configurations there